pom.xml

```diff
@@ -45,10 +45,16 @@
       <artifactId>joda-time</artifactId>
       <version>2.10.6</version>
     </dependency>
+    <dependency>
```

Comment:
Please could this be moved to be with the other test dependencies.

Reply:
Done